src/hooks/useSuggestions.js

```diff
@@ -187,6 +210,48 @@ const useSuggestions = () => {
     return true;
   };
 
+  const filterCurrentUser = arrayData => arrayData.filter(({ id }) => id !== myId);
```

Comment:
i would change the name of the arrayData parameter to something more clear

Reply:
I can call it votes since I call this with negativeVotes and positiveVotes.